src/public/client.ts

```diff
@@ -1,24 +1,57 @@
-/* global helper */
+/* global */
+//TODO: Move this function into separate helper function
+type Mapping = {
+  domain: string
+  ip: string
+  port: string
+}
+
+const getElement = (query: string, root?: HTMLElement): HTMLElement => {
+  if (!root) {
+    return (
+      document.querySelector<HTMLElement>(query) ||
+      (document.createElement('div') as HTMLElement)
+    )
+  }
+  return (
+    root.querySelector(query) || (document.createElement('div') as HTMLElement)
+  )
+}
 
-const create: HTMLElement = helper.getElement('.create')
-const hostSelector: HTMLElement = helper.getElement('.hostSelector')
+const create: HTMLElement = getElement('.create')
+const hostSelector: HTMLElement = getElement('.hostSelector')
+const domainList: HTMLElement = getElement('.domainList')
+const dropDownSubDomains: HTMLElement = getElement('.dropdown-menu')
```

Comment:
`domainDropdown`. The dropdown shows domains, not subdomains. Here's how the url is structured:

`https(protocol)://(subdomain).(domainname).(topLevelDomain)/(pathname)?(queryparameters)`

Reply:
`subdomain` is the inputbox, the dropdown is actually domainname